### PR TITLE
close and comment submit buttons

### PR DESCRIPTION
**Pull request by eliasow**
3 hours

### CODE REVIEW

File: frontend/src/ListView.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 import React, { Component } from "react";
+import Home from "./Home";
+import ReactDOM from "react-dom";
+
+function goHome(){//add browser router??
```
Comment (davidayu) [inline frontend/src/ListView.js:5]: remove comment. will make note of this in github

**Comment by davidayu** [comment]
Looks good to me

**Review by ksittner — COMMENTED** [review]
Looks good now

**Review by ksittner — APPROVED** [review]
Looks good now

**Comment by AmeyaMalekar** [comment]
Looks good to me!

File: frontend/src/ListView.js (hunk 1 above)
Comment (eliasow) [reply frontend/src/ListView.js:5]: good idea

**Comment by noahmarxx** [comment]
Good work Elias

**Comment by eliasow** [comment]
> Looks good now

Thank you!

**Comment by eliasow** [comment]
> Looks good to me

Thank you for reviewing

**Comment by eliasow** [comment]
> Good work Elias

Thank you Noah

**Pull request merged**